efficientdet/keras/utils_keras.py

```diff
@@ -1,25 +1,44 @@
+# Lint as: python3
+# Copyright 2020 Google Research. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Common keras utils."""
+# gtype import
+
 from typing import Text
 import tensorflow as tf
+import utils
 
 
-def batch_normalization(is_training_bn: bool,
-                        init_zero: bool = False,
-                        data_format: Text = 'channels_last',
-                        momentum: float = 0.99,
-                        epsilon: float = 1e-3,
-                        strategy: Text = None,
-                        name: Text = 'tpu_batch_normalization'):
-  """Performs a batch normalization followed by a non-linear activation.
+def build_batch_norm(is_training_bn: bool,
```

Comment:
I changed batch_normalization to build_batch_norm, because batch_normalization reads like a function that takes a tensor as input and run the batch norm, but here it actually just build the batch_norm_layer.